readthedocsext/theme/templates/projects/import_base.html

```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
                   {% if wizard.steps.prev %}
                     <button class="ui button"
                             name="wizard_goto_step"
-                            value="{{ wizard.steps.prev }}">{% trans "Previous" %}</button>
+                            value="{{ wizard.steps.prev }}">
+                      {% trans "Previous" %}
+                    </button>
                   {% endif %}
 
                   {% if wizard.steps.next %}
                     <input class="ui button primary" type="submit" value="{% trans "Next" %}" />
-                  {% else %}
+                  {% endif %}
+
+                  {% if wizard.steps.current == "config" %}
+                    <div class="ui small modal" data-modal-id="i-need-help">
+                      <div class="header">Add configuration file to your project</div>
```

Comment:
All the strings here are missing localization

```suggestion
                      <div class="header">{% trans "Add a configuration file to your project" %}</div>
```